chapters/ch02.md

```diff
@@ -2,37 +2,49 @@
 
 ->
 
-
+# Tại Sao Cần Có Chiến Lược Machine Learning?
 
 After finishing this book, you will have a deep understanding of how to set technical direction for a machine learning project.
 ->
 
+Sau khi đọc xong cuốn sách này, bạn sẽ hiểu sâu hơn về cách lựa chọn hướng giải quyết kỹ thuật cho đề tài machine learning. 
+
 
 
 But your teammates might not understand why you’re recommending a particular direction.
 ->
 
+Nhưng có thể cộng sự của bạn có thể sẽ không khỏi thắc mắc tại sao bạn lại chọn hướng đi như vậy. 
+
 
 
 Perhaps you want your team to define a single-number evaluation metric, but they aren’t convinced. How do you persuade them?
 ->
 
+Có thể bạn muốn cả đội xác định ra và dùng một phép đo duy nhất, nhưng điều đó không hoàn toàn thuyết phục tất cả mọi người. Bạn sẽ làm sao để thuyết phục họ?
```

Comment:
Thêm 'evaluation metric' vào glossary.

Chỗ 'a single-number evaluation metric' tớ nghĩ dịch là '... định nghĩa một phép đánh giá bằng một con số'. Tuy nhiên, mình có thể chờ thêm đóng góp.

'but they aren't convinced' -> 'nhưng họ không thấy thuyết phục'.

Reply:
Đồng ý với @aivivn là "'a single-number evaluation metric" ý là phép đo mà nó chỉ đưa ra 1 số duy nhất để đánh giá classifier của mình. Mình không rõ có quyền để viết thêm vào không (hoặc có thể giải thích trong glossary)nhưng mình có thể đưa ra ví dụ như ở đây:

> It is always better to establish as a data scientist a single-number evaluation metric for your team to optimize. Accuracy is an example of a single-number evaluation metric​ and allows you quickly to compare two classifiers.
If classifier A obtains 97% accuracy, and classifier B obtains 90% accuracy, then we judge classifier A to be superior. In contrast, Precision and Recall is not a single-number evaluation metric: It gives two 2 numbers for assessing your classifier. Suppose your algorithms perform as follows: Classifier Precision Recall A 95% 90% B 98% 85% , neither classifier is obviously superior, so it doesn’t immediately guide you toward picking one.

Nguồn: https://towardsdatascience.com/how-to-select-the-right-evaluation-metric-for-machine-learning-models-part-3-classification-3eac420ec991

Hoặc cũng có thể dùng nguồn là chính video giảng về "Single Number Evaluation Metric" của Adrew Ng

Reply:
Mình cũng vừa mới đọc chap này. Đang update glossary